### PR TITLE
Use the slice mapper to render 2D volumes

### PR DESCRIPTION
**Pull request by boucaud**
See https://github.com/Kitware/vtk-js/pull/1605
@floryst

### CODE REVIEW

File: src/vtk/CustomVolumeRepresentationProxy/index.js [hunk 1]
```diff
@@ -58,6 +58,14 @@ function vtkCustomVolumeRepresentationProxy(publicAPI, model) {
       publicAPI.setColorBy(...colorBy);
     }
   };
+
+  publicAPI.setIs2DVolume = (is2D) => {
+    superClass.setIs2DVolume(is2D);
+    if (is2D) {
```
Comment (floryst) [inline src/vtk/CustomVolumeRepresentationProxy/index.js:64]: Will there ever be a chance that `setIs2DVolume(false)` is called, and if so, what should happen?

Also, in setting `sliceUseColorByFor{Color|Opacity}`, I think calling `publicAPI.setColorBy(...publicAPI.getColorBy)` at the end would be desired. (See `setSliceUseColorByForColor` above.)

Comment (boucaud) [reply src/vtk/CustomVolumeRepresentationProxy/index.js:64]: > Will there ever be a chance that setIs2DVolume(false) is called, and if so, what should happen?

I don't think we need to do anything in this case. `setIs2DVolume(false)` doesn't directly change any actor's visibility.

**Comment by floryst** [comment]
LGTM

**Pull request merged**